gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/rest/api/service/impl/JsonSchemaServiceTest.java

```diff
@@ -48,10 +48,10 @@ public void shouldRejectInValidJsonConfigurationWithDependencies() throws IOExce
                 "#/ssl/trustStore: required key [content] not found\n" +
                 "#/ssl/trustStore/type: \n" +
                 "#/ssl/trustStore: required key [path] not found\n" +
-                "#/ssl/trustStore: required key [path] not found\n" +
+                "#/ssl/trustStore: required key [content] not found\n" +
```

Comment:
Why did this test change?

Reply:
This test is flaky.
More times it passes with "content" instead of "path

Reply:
Removed